src/support/z_file_locate.erl

```diff
@@ -234,7 +235,7 @@ generate_preview(true, _Mime, Path, OriginalFile, Filters, Medium, Context) ->
             PreviewFilePath = filename:join(PreviewDir, Path),
             case z_media_preview:convert(z_convert:to_list(Filename), OriginalFile, z_convert:to_list(PreviewFilePath), Filters, Context) of
```

Comment:
Line 236 is too long:             case z_media_preview:convert(z_convert:to_list(Filename), OriginalFile, z_convert:to_list(PreviewFilePath), Filters, Context) of.